ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -93,8 +99,12 @@ message GetPartiesResponse {
   repeated PartyDetails party_details = 1;
 }
 
-// Required authorization: ``HasRight(ParticipantAdmin)``
+// Required authorization: ``HasRight(ParticipantAdmin) OR IsAuthenticatedIdentityProviderAdmin(identity_provider_id)``
 message ListKnownPartiesRequest {
+
+  // The id of the ``Identity Provider`` whose parties should be retrieved.
+  // Optional, if not set, assume the party is managed by the default identity provider or party is not hosted by the participant.
```

Comment:
What does it mean that the party is assumed to be managed by the default identity provider? I.e. how does it affect the response?
Similarly, what are the implications for assuming a party is not hosted by this participant? Will it be included in the response?

Reply:
`assume the party` - which party is this?